target/classes/application.properties

```diff
@@ -0,0 +1,16 @@
+# SPRING
```

Comment:
should target directory be in gitignore?

Reply:
You are right, it should, i forgot to add it.